pkg/manifests/types.go

```diff
@@ -388,6 +391,25 @@ type NodeExporterCollectorBuddyInfoConfig struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
+// The `NodeExporterCollectorSystemdConfig` resource works as an on/off switch for
+// the `systemd` collector of the `node-exporter` agent.
+// By default, the `systemd` collector is disabled.
+// If enabled, the following metrics become available:
+// `node_systemd_socket_accepted_connections_total`,
+// `node_systemd_socket_current_connections`,
+// `node_systemd_socket_refused_connections_total`,
+// `node_systemd_system_running`,
+// `node_systemd_timer_last_trigger_seconds`,
+// `node_systemd_unit_state`,
+// `node_systemd_units`,
+// `node_systemd_version`.
+// Among them, the `node_systemd_unit_state` metric is the most useful show the state of each  systemd unit. So its cardinality cound be high.
```

Comment:
```suggestion
// Of these metrics, the `node_systemd_unit_state` metric is the most useful because it shows the state of each `systemd` unit. However, note that the cardinality for this metric might be high.
```